docs/source/markdown/podman-create.1.md

```diff
@@ -1219,6 +1219,15 @@ Without this argument the command will be run as root in the container.
 
 Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.
 
+Rootless user --userns=Key mappings:
+
+Key       | Host User |  Container User
+----------|---------------|---------------------
+""        |$UID           |0 (Default User account mapped to root user in container)
+keep-id   |$UID           |$UID (Map user account to same UID within containe.)
+auto      |$UID           | nil (Host User UID is not mapped into container)
+nomap     |$UID           | nil (Host User UID is not mapped into container)
```

Comment:
You have an ending period '.' for keep-id within the parens, which I prefer.  Your pick, but you should keep them consistent.